service/src/test/java/org/sunbird/learner/actors/UserManagementActorTest.java

```diff
@@ -1,1770 +1,335 @@
+    when(userService.getRootOrgIdFromChannel(Mockito.anyString())).thenReturn("rootOrgId");
 
-  @Test
-  public void testBCreateUserInfoWithInvalidRole() {
-    TestKit probe = new TestKit(system);
-    ActorRef subject = system.actorOf(props);
+    PowerMockito.mockStatic(ElasticSearchUtil.class);
+    when(ElasticSearchUtil.getDataByIdentifier(
+            Mockito.anyString(), Mockito.anyString(), Mockito.anyString()))
+        .thenReturn(map);
 
-    Request reqObj = new Request();
-    reqObj.setOperation(ActorOperations.CREATE_USER.getValue());
-    Map<String, Object> innerMap = new HashMap<>();
-    innerMap.put(JsonKey.USERNAME, "sunbird_dummy_user_4141411");
-    innerMap.put(JsonKey.EMAIL, "sunbird_dummy_user_4141411@gmail.com");
-    innerMap.put(JsonKey.PASSWORD, "password");
-    // Add Roles
-    List<String> roleList = new ArrayList<>();
-    roleList.add("CONTENT_CURATION_1");
-    roleList.add("CONTENT_CREATION_2");
-    roleList.add("MEMBERSHIP_MANAGEMENT_3");
-    innerMap.put(JsonKey.ROLES, roleList);
-    Map<String, Object> request = new HashMap<String, Object>();
-    request.put(JsonKey.USER, innerMap);
-    reqObj.setRequest(request);
+    PowerMockito.mockStatic(Util.class);
+    Util.getUserProfileConfig(Mockito.any(ActorRef.class));
 
-    subject.tell(reqObj, probe.getRef());
-    ProjectCommonException exc =
-        probe.expectMsgClass(duration("200 second"), ProjectCommonException.class);
-    assertTrue(null != exc);
-  }
+    PowerMockito.mockStatic(UserUtil.class);
+    UserUtil.setUserDefaultValue(Mockito.anyMap(), Mockito.anyString());
 
-  @Test
-  public void testCgetUserAddressInfo() {
-    String encUserId = userId;
-    if ("ON".equalsIgnoreCase(encryption)) {
-      try {
-        encUserId = encryptionService.encryptData(encUserId);
-      } catch (Exception e) {
-        // TODO Auto-generated catch block
-        ProjectLogger.log(e.getMessage(), e);
-      }
-    }
-    Response response =
-        operation.getRecordsByProperty(
-            addressDB.getKeySpace(), addressDB.getTableName(), JsonKey.USER_ID, encUserId);
-    Map<String, Object> result = (Map<String, Object>) (response.getResult());
-    List<Map<String, Object>> addressList =
-        (List<Map<String, Object>>) result.get(JsonKey.RESPONSE);
-    userAddrIdToDelete = (String) ((Map<String, Object>) addressList.get(0)).get(JsonKey.ID);
-    assertEquals(addressList.size(), 2);
+    Map<String, Object> requestMap = new HashMap<>();
+    requestMap.put(JsonKey.TNC_ACCEPTED_ON, 12345678L);
+    when(UserUtil.encryptUserData(Mockito.anyMap())).thenReturn(requestMap);
   }
 
   @Test
-  public void testCgetUserAddressInfoAndDelete() {
-
-    String encUserId = userId;
-    if ("ON".equalsIgnoreCase(encryption)) {
-      try {
-        encUserId = encryptionService.encryptData(encUserId);
-      } catch (Exception e) {
-        ProjectLogger.log(e.getMessage(), e);
-      }
-    }
+  public void testCreateuserSuccessWithCallerId() {
```

Comment:
rename test-case accordingly
testCreateUserSuccessWith???
what is caller id here